feature/settings/src/main/kotlin/org/meshtastic/feature/settings/radio/component/NetworkConfigItemList.kt

```diff
@@ -110,6 +110,41 @@ fun NetworkConfigScreen(viewModel: RadioConfigViewModel = hiltViewModel(), onBac
             viewModel.setConfig(config)
         },
     ) {
+        // Display device connection status
+        state.deviceConnectionStatus?.let { connectionStatus ->
+            if (
+                connectionStatus.wifi?.status?.isConnected == true ||
+                connectionStatus.ethernet?.status?.isConnected == true
+            ) {
+                item {
+                    TitledCard(title = stringResource(R.string.connection_status)) {
+                        connectionStatus.wifi?.let { wifiStatus ->
+                            if (wifiStatus.status.isConnected) {
+                                Text(
```

Comment:
I'm assuming only one of these will ever show. If that's the case, maybe just get the text first and use a single composable to display.

Using `org.meshtastic.core.ui.component.ListItem` with a null onClick would match you to the other rows on this screen.

Reply:
I'd assume it can do both, but I've not had anything that has ethernet so I haven't looked at how firmware handles it